vagrant/dev/Vagrantfile

```diff
@@ -19,4 +19,33 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   if Vagrant.has_plugin?("vagrant-cachier")
     config.cache.scope = :box
   end
+
+  # This allows the VM to be restarted during provisioning, necessary after
+  # updating the kernel
+  unless Vagrant.has_plugin?("vagrant-reload")
+    raise "vagrant-reload plugin is not installed"
+  end
+
+=begin
+  With some versions of VirtualBox, the following error may occur when running
+  vagrant up.
```

Comment:
This perhaps has nothing to do with versions of VirtualBox, but rather the fact that we are updating the kernel.